### PR TITLE
Remove default form [MAILPOET-3238]

### PR DESCRIPTION
**Pull request by costasovo**
[MAILPOET-3238]

[MAILPOET-3238]: https://mailpoet.atlassian.net/browse/MAILPOET-3238

### CODE REVIEW

File: assets/js/src/forms/heading.jsx [hunk 1]
```diff
@@ -5,35 +5,33 @@ import MailPoet from 'mailpoet';
...
-        >
-          {plusIcon}
-          <span>{MailPoet.I18n.t('new')}</span>
-        </Link>
-      </TopBarWithBeamer>
-      <h1 className="mailpoet-newsletter-listing-heading-empty title">{' '}</h1>
-    </>
-  );
+export const goToSelectTemplate = () => {
+  setTimeout(() => {
```
Comment (pavel-mailpoet) [inline assets/js/src/forms/heading.jsx:9]: Do we need the timeout here? I don't see any events here mentioned in the comment, or should we add an event?

Comment (costasovo) [reply assets/js/src/forms/heading.jsx:9]: Hmm, good question. I just moved the code. I can't see any event even before the change. I'll try to find out what happened to it.

Comment (pavel-mailpoet) [reply assets/js/src/forms/heading.jsx:9]: I **think** we used to have a Mixpanel event for tracking a form creation. That may have happened here.

Comment (costasovo) [reply assets/js/src/forms/heading.jsx:9]: Hmm, the event is gone :( I'll add it to template selection function.

**Review by pavel-mailpoet — DISMISSED** [review]

**Review by veljkho — APPROVED** [review]

**Pull request merged**